website/src/pages/universal-code-search.tsx

```diff
@@ -0,0 +1,279 @@
+import { Link } from 'gatsby'
+import * as React from 'react'
+import { Blockquote } from '../components/Blockquote'
+import { ContentPage } from '../components/content/ContentPage'
+import { ContentSection } from '../components/content/ContentSection'
+import { Jumbotron } from '../components/Jumbotron'
+import Layout from '../components/Layout'
+import { CustomerLogosSection } from '../components/product/CustomerLogosSection'
+import { GetSourcegraphNowActions } from '../css/components/actions/GetSourcegraphNowActions'
+
+export default ((props: any) => (
+    <Layout location={props.location}>
+        <ContentPage
+            title="What is a Universal Code Search?"
```

Comment:
```suggestion
            title="What is Universal Code Search?"
```

might need to fix elsewhere too